rpm/rpm.go

```diff
@@ -291,8 +290,9 @@ func CombinedProber(
 					probeCount+1,
 				)
 			}
-			transport := http2.Transport{}
+			transport := &http.Transport{}
```

Comment:
I assume that you are comfortable that the use of the `http2.ConfigureTransport` call below will do the right thing and that we do not need to manually specify that we want HTTP2? It seems like your change is absolutely correct, I just want to verify that I am reading it correctly!

Reply:
That's my understanding and I'm comfortable with it.

Reply:
Great! Again, I agree and just wanted to make sure that we were on the same page!!